client/state/selectors/get-inline-help-support-variation.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
```

Comment:
The file name could be improved, perhaps just get-support-variation would be fitting.

Reply:
Yes, I'd prefer the latter. Not a blocker thought / fine to do in a follow-up PR.